R/IDateTime.R

```diff
@@ -87,8 +87,9 @@ round.IDate = function (x, digits=c("weeks", "months", "quarters", "years"), ...
 `+.IDate` = function (e1, e2) {
   if (nargs() == 1L)
     return(e1)
+  # TODO: investigate Ops.IDate method a la Ops.difftime
```

Comment:
As noted in `?Ops`

> The classes of both arguments are considered in dispatching any member of this group. For each argument its vector of classes is examined to see if there is a matching specific (preferred) or Ops method. If a method is found for just one argument or the same method is found for both, it is used. If different methods are found, there is a warning about ‘incompatible methods’: in that case or if no method is found for either argument the internal method is used.

Hence we can't really reach this branch since the `incompatible methods` barrier is hit first.